bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,95 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/products/lockwise" %}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_favicon %}{{ static('img/firefox/lockwise/favicon.ico') }}{% endblock %}
+
+{% block page_title %}{{_('Firefox Lockwise — password manager —  take your passwords everywhere')}}{% endblock %}
+
+{% block page_desc %}{{ _('Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('lockwise') }}
+{% endblock %}
+
+{% block body_id %}firefox-lockwise{% endblock %}
+{% block body_class %}firefox-lockwise{% endblock %}
+
+{% block content %}
+  <main role="main">
+    {% if LANG.startswith('en-') %}
+      {% set _img = 'firefox/lockwise/en.png'%}
+    {% elif LANG == 'de' %}
+      {% set _img = 'firefox/lockwise/de.png'%}
+    {% elif LANG == 'fr' %}
+      {% set _img = 'firefox/lockwise/fr.png'%}
+    {% elif LANG == 'it' %}
+      {% set _img = 'firefox/lockwise/it.png'%}
+    {% elif LANG.startswith('es-') %}
+      {% set _img = 'firefox/lockwise/es.png'%}
+    {% else %}
+      {% set _img = 'firefox/lockwise/en.png'%}
+    {% endif %}
+    {% call hero(
+      title='Firefox Lockwise',
+      subheading=_('Take your passwords everywhere'),
+      desc=_('Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.'),
+      class='mzp-c-hero mzp-has-image lockwise-hero',
+      image_url= _img,
+      include_cta=True,
+      heading_level=1,
+      subheading_level=4,
```

Comment:
Nit: This should probably be a level 2 as it follows an h1. You may need to set a custom heading size in your CSS to compensate.